grafanalib/core.py

```diff
@@ -714,6 +720,7 @@ class Dashboard(object):
     )
     timezone = attr.ib(default=UTC)
     version = attr.ib(default=0)
+    graphTooltip = attr.ib(default=None, validator=instance_of(int))
```

Comment:
Why is `graphTooltip` an `int`?

Reply:
In the JSON dashboards I'm porting, graphTooltip is an int.

Reply:
No, I don't. Just asking.